src/lib/yang/parser.lua

```diff
@@ -189,7 +204,18 @@ end
 function Parser:parse_string()
    if self:check("'") then return self:parse_qstring("'")
    elseif self:check('"') then return self:parse_qstring('"')
-   else return self:take_while("[^%s;{}\"'/]") end
+   else
+      local ret = ""
+      repeat
+         ret = ret .. self:take_while("[^%s;{}\"'/]")
+         if self:is_eof() then break end
+         if self:peek_n(2) == "/*" then break end
+         if self:check("/") then
+            ret = ret .. "/"
+         end
+      until not self:check_pattern("[^%s;{}\"'/]")
+      return ret
```

Comment:
Since parse_string is on some hot paths, we should probably fix the quadratic algorithm before releasing; i.e. local ret = [], then table.insert the pieces as we parse them, then return `return table.concat(ret)`